rotkehlchen/exchanges/exchange.py

```diff
@@ -18,6 +20,7 @@
 log = RotkehlchenLogsAdapter(logger)
 
 
+ExchangeQueryBalances = Tuple[Optional[Dict[Asset, Balance]], str]
```

Comment:
If you don't like `ExchangeQueryBalances` I can rename it.

Reply:
totally fine!